go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/talos-systems/talos
 
-go 1.13
+go 1.16
```

Comment:
one thing which is not clear for me from the docs -- does it allow module to be imported with Go 1.15 now?

this is mostly important for machinery package which provides API to access the cluster or generate configs

Reply:
Yes. Currently, the code will still be imported and compiled, and the error message will include a Go version mismatch only if the code will fail to compile.